easybuild/easyblocks/generic/intelbase.py

```diff
@@ -112,6 +112,7 @@ def extra_options(extra_vars=None):
         extra_vars = EasyBlock.extra_options(extra_vars)
         extra_vars.update({
             'license_activation': [ACTIVATION_LIC_SERVER, "License activation type", CUSTOM],
+            'licensed': [True, "Set to False if Intel software does not require a runtime licence", CUSTOM],
```

Comment:
Should I just be leveraging the existing `license_activation` variable, repurposing it to also handle a False value.